apps/backend/.env.example

```
@@ -4,4 +4,7 @@ USE_OPENAI_MODEL_NAME=
 
 # Better-auth
 BETTER_AUTH_SECRET=
-BETTER_AUTH_URL=http://localhost:8787 # Base URL of your app
\ No newline at end of file
+BETTER_AUTH_URL=http://localhost:8787 # Base URL of your app
+
+# GitHub MCP
+GITHUB_MCP_TOKEN=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)